bullyBot.py

```diff
@@ -0,0 +1,47 @@
+#bullyBot.py
+import os
+import discord
+from discord.ext import commands
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Why do we need to load dot environment? I don't see the use of any environment variables or anything. Check if this bot still works without this.

Reply:
My bot token lives in an environment variable.

Reply:
Makes sense +1